src/components/App/App.tsx

```diff
@@ -1,21 +1,104 @@
-import React from 'react';
-import { withTranslation, WithTranslation } from '@wix/wix-i18n-config';
-import s from './App.scss';
+const validator = require('validator');
 
-interface AppProps extends WithTranslation {}
+import './App.scss';
+import React, { useState, useCallback } from 'react';
+import {
+  Layout,
+  Cell,
+  Breadcrumbs,
+  WixStyleReactProvider,
+  Page,
+} from 'wix-style-react';
+import ActiveBar from '../ActiveBar/ActiveBar';
+import GeneralInfoForm from '../GeneralInfoForm/GeneralInfoForm';
+import RoleDetails from '../RoleDetails/RoleDetails';
+import SavedData from '../SavedData/SavedData';
+import { getColorById } from '../colorOptions';
 
-class App extends React.Component<AppProps> {
-  render() {
-    const { t } = this.props;
+const initData = { firstname: '', lastname: '', color: '' };
```

Comment:
Maybe you can be more specific ?
```suggestion
const initialFormData = { firstname: '', lastname: '', color: '' };
```